homeassistant/components/tts/__init__.py

```diff
@@ -619,6 +620,16 @@ async def post(self, request: web.Request) -> web.Response:
         base = self.tts.base_url or get_url(self.tts.hass)
         url = base + path
 
+        hass.bus.fire(
+            DOMAIN,
```

Comment:
This parameter is supposed to be the type of event. You're just putting the domain of the component which will make it very tough to use since its so generic. Would recommend setting the type to something which describes what is happening. See here: https://developers.home-assistant.io/docs/dev_101_events/#firing-events